src/core/zowe/core_for_zowe_sdk/profile_manager.py

```diff
@@ -174,7 +177,27 @@ def get_profile(
         cfg_profile = Profile()
         try:
             cfg_profile = cfg.get_profile(
-                profile_name=profile_name, profile_type=profile_type
+                profile_name=profile_name, profile_type=profile_type, config_type=config_type, validate_schema=validate_schema
+            )
+        except jsonschema.exceptions.ValidationError as exc:
+            raise jsonschema.exceptions.ValidationError(
+                f"Instance was invalid under the provided $schema property, {exc}"
+            )
+        except jsonschema.exceptions.SchemaError as exc:
+            raise jsonschema.exceptions.SchemaError(
+                f"The provided schema is invalid, {exc}"
+            )
+        except jsonschema.exceptions.UndefinedTypeCheck as exc:
+            raise jsonschema.exceptions.UndefinedTypeCheck(
+                f"A type checker was asked to check a type it did not have registered, {exc}"
+            )
+        except jsonschema.exceptions.UnknownType as exc:
+            raise jsonschema.exceptions.UnknownType(
+                f"Unknown type is found in schema_json, exc"
+            )
+        except jsonschema.exceptions.FormatError as exc:
+            raise jsonschema.exceptions.FormatError(
+                f"Validating a format config_json failed for schema_json, {exc}"
             )
```

Comment:
I think the exception handling should be moved to the `validate_schema` method in `config_file.py`. This would ensure consist error messages whether the schema is validated through the ConfigFile or ProfileManager class.

Reply:
I tried moving it to the `config.py` but it doesn't raise error it skips the profile.